### PR TITLE
Change names of outputs to __return_{i}.

### PR DESCRIPTION
**Pull request by orausch**
This is preparatory work for parsing full training steps.

### CODE REVIEW

File: daceml/onnx/onnx_importer.py [hunk 1]
```diff
@@ -298,6 +301,32 @@ def __init__(self,
...
+
+            copy_out_state.add_edge(copy_out_state.add_read(clean_name), None,
+                                    copy_out_state.add_write(new_output_name),
+                                    None,
+                                    self.sdfg.make_array_memlet(clean_name))
+
+        # finally, rename outputs
+        self.outputs = new_output_names
+
+        self.sdfg.apply_transformations_repeated(interstate.StateFusion)
```
Comment (tbennun) [inline daceml/onnx/onnx_importer.py:328]: Why do you need to do this? Also, there's a faster, safer FuseStates pass

**Review by tbennun — CHANGES_REQUESTED** [review]
I like everything in this PR. One comment about fusing states though.

File: daceml/onnx/onnx_importer.py (hunk 1 above)
Comment (orausch) [reply daceml/onnx/onnx_importer.py]: Switched to `fuse_states`. I can't use the `FuseStates` pass since daceml is still on `0.13.3`

**Review by tbennun — APPROVED** [review]
LGTM, but what's up with the test?

**Pull request merged**